fab/environments.yml

```diff
@@ -165,34 +165,37 @@ icds:
         num_processes: 1
       CaseSearchToElasticsearchPillow:
         num_processes: 1
-      CaseToElasticsearchPillow:
-        num_processes: 1
```

Comment:
Whoa. I think we should only remove `CaseSearchToElasticSearchPillow` If we remove `CaseToElasticsearchPillow` exports or reports such as case list won't work as they are dependant on that